cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -386,6 +402,7 @@ public static void main(String[] args) {
         System.err.println("Failed to start Standalone CDAP");
         e.printStackTrace(System.err);
       }
+      main.stopExternalProcesses();
       Runtime.getRuntime().halt(-2);
```

Comment:
Not quite sure what -2 means here. We just need to make sure it is non-zero to indicate abnormal process termination. Does our scripts look for these values?